woocommerce/woocommerce-ios
# Log in with Site URL

Closes #263. In this PR, the Login via Site URL flow is available. After the site URL has been validated (is a WP.com site or self-hosted with Jetpack), the user can go to a custom flow where they sign into their WP.com account using a username / password combo.

This PR is the companion PR to https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/71.

## To test
1. Make sure you're logged out of the WCiOS app before starting this test.
2. Check out this PR.
3. Start the login process by selecting "Login with Jetpack" button
4. Select the "Log in by entering your site address" button
5. Log in with a WordPress site that doesn't have Jetpack connected, ex: https://thuycopeland.com
6. An error should appear, "Looks like your site isn't set up to use this app. Make sure your site has Jetpack installed to continue."
7. Delete the address and log in with a self-hosted woocommerce store that is connected using Jetpack.
8. If the site validates as hasJetpack, you will proceed to the next screen.
9. Log in to the site by using the associated WP.com username and password.
10. After a successful login, the site picker attempts to match the siteURL string. The site you typed in should be pre-selected. (Continue)
11. Expected behavior: you should land on the My store tab.

## Known issues
You may observe some missing icons while testing the different login flows. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/67 introduced a bundle loading issue for Woo and has been reported. I won't merge the PR until the bug https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/38 has been fixed first.

Update release notes:
